packages/cardano/README.md

```
@@ -0,0 +1,973 @@
+- [Installation](#installation)
+- [Quick Start](#quick-start)
+- [API Reference](#api-reference)
+  - [cardano()](#cardano)
+  - [getValidators](#getvalidators)
+  - [getDelegations](#getdelegations)
+  - [getBalances](#getbalances)
+  - [getNonce](#getnonce)
+  - [estimateFee](#estimatefee)
+  - [sign](#sign)
+  - [preHash / compile](#prehash--compile)
+  - [broadcast](#broadcast)
```

markdownlint-cli2 (0.22.0)

[warning] Link fragments should be valid

(MD051, link-fragments)